integration-tests/deployment/address_book.go

```diff
@@ -109,9 +110,6 @@ func (m *AddressBookMap) Save(chainSelector uint64, address string, typeAndVersi
 		// First time chain add, create map
 		m.AddressesByChain[chainSelector] = make(map[string]TypeAndVersion)
 	}
-	if _, exists := m.AddressesByChain[chainSelector][address]; exists {
```

Comment:
@AnieeG @connorwstein  i needed to change this. i think it's safe since it's normal for merge to overwrite and existing addr. problem was uncovered if i wanted to re-run a migration in `chainlink-deployments`